javascript-forms/form-events.js

```diff
@@ -0,0 +1,40 @@
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log('event.target.name', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log('event.target.name', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('value of', event.target.name, event.target.value);
+}
+
+var $input0 = document.forms[0].elements[0];
+var $input1 = document.forms[0].elements[1];
```

Comment:
You should use `querySelector`, don't make your element selections dependent on the order of elements in the document.